java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java

```diff
@@ -668,7 +670,9 @@ public void testMapNullValueIsRejected() throws Exception {
               + "  \"int32ToMessageMap\": {\"2\": null}\n"
               + "}",
           builder);
-      fail();
+      assertEquals(0, builder.getInt32ToEnumMapMap().size());
```

Comment:
Please don't change this test for non-enum map values. As I suggested above, add a new method instead (do not split the code from the existing "testMapNullValueIsRejected()" method.